.eslintrc

```diff
@@ -2,7 +2,34 @@
     "extends": "airbnb",
     "rules": {
         // Override any settings from the airbnb configuration
-        "indent": [2, 4],
-        "comma-dangle": 0
+        // "indent": [2, 4],
```

Comment:
Nah, I commented it out (and turned it off later) because our current code doesn't match this style.